dojo.py

```diff
@@ -241,23 +239,22 @@ def allocate_room(self, person):
                 #return 'Fellow allocated both living space and office'
             else:
                 fellow_single_allocation = {}
-                fellow_single_allocation[person.full_name] = \
-                    self.offices['available'][
+                fellow_single_allocation['name'] = person.full_name
+                fellow_single_allocation['office'] = self.offices['available'][
                     randint(0, (len(self.offices['available']) - 1))]
                 self.fellow_allocations.append(fellow_single_allocation)
                 for room in self.rooms:
-                    if room.room_name == \
-                            fellow_single_allocation[person.full_name]:
+                    if room.room_name == fellow_single_allocation['office']:
```

Comment:
you can check multiple conditions in one if block to reduce nesting if the outermost block doesnt have an else
i.e. instead of:
`
if(condition1):
`
`
 if(condition2):
`
you can do:
`if(condition1 && condition2):`

Reply:
great idea, let me refactor that